tests/simulator/test_advance.py

```
@@ -5,6 +5,9 @@
 import numpy as np
 from ddt import ddt
 
+import pyphare.core.box as boxm
+from pyphare.core.box import amr_to_local
+
 
 from pyphare import cpp
 import pyphare.core.box as boxm
```

Ruff (0.15.2)

[error] Redefinition of unused `boxm` from line 8: `boxm` redefined here

Remove definition: `boxm`

(F811)